addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/CommandInjectionScanRule.java

```diff
@@ -350,6 +350,11 @@ public int getRisk() {
         return Alert.RISK_HIGH;
     }
 
+    private String getOtherInfo(String testType, String testValue) {
+        return String.format(
+                Constant.messages.getString(MESSAGE_PREFIX + "otherinfo." + testType), testValue);
+    }
+
```

Comment:
```suggestion
    private String getOtherInfo(String testType, String testValue) {
        return Constant.messages.getString(MESSAGE_PREFIX + "otherinfo." + testType), testValue);
    }

```

Reply:
Probably needs a spotlessApply after that.

Reply:
`testType),` → `testType,`

Reply:
AHHHHHHHH! Thanks!

Reply:
oops, thanks @thc202